python/srf/core/CMakeLists.txt

```diff
@@ -13,13 +13,37 @@
 # the License.
 # =============================================================================
 
-srf_add_pybind11_module(NAME common SOURCE common.cpp)
-srf_add_pybind11_module(NAME executor SOURCE executor.cpp)
-srf_add_pybind11_module(NAME logging SOURCE logging.cpp)
-srf_add_pybind11_module(NAME node SOURCE node.cpp)
-srf_add_pybind11_module(NAME operators SOURCE operators.cpp)
-srf_add_pybind11_module(NAME options SOURCE options.cpp)
-srf_add_pybind11_module(NAME pipeline SOURCE pipeline.cpp)
-srf_add_pybind11_module(NAME segment SOURCE segment.cpp)
-srf_add_pybind11_module(NAME segment_definition SOURCE segment_definition.cpp)
-srf_add_pybind11_module(NAME subscriber SOURCE subscriber.cpp)
+# srf_add_pybind11_module(NAME common SOURCE common.cpp)
+# srf_add_pybind11_module(NAME executor SOURCE executor.cpp)
+# srf_add_pybind11_module(NAME logging SOURCE logging.cpp)
+# srf_add_pybind11_module(NAME node SOURCE node.cpp)
+# srf_add_pybind11_module(NAME operators SOURCE operators.cpp)
+# srf_add_pybind11_module(NAME options SOURCE options.cpp)
+# srf_add_pybind11_module(NAME pipeline SOURCE pipeline.cpp)
+# srf_add_pybind11_module(NAME segment SOURCE segment.cpp)
+# srf_add_pybind11_module(NAME segment_definition SOURCE segment_definition.cpp)
+# srf_add_pybind11_module(NAME subscriber SOURCE subscriber.cpp)
+
+add_pybind11_module(common SOURCE_FILES common.cpp LINK_TARGETS pysrf OUTPUT_TARGET common_target)
+add_pybind11_module(executor SOURCE_FILES executor.cpp LINK_TARGETS pysrf OUTPUT_TARGET executor_target)
+add_pybind11_module(logging SOURCE_FILES logging.cpp LINK_TARGETS pysrf OUTPUT_TARGET logging_target)
+add_pybind11_module(node SOURCE_FILES node.cpp LINK_TARGETS pysrf OUTPUT_TARGET node_target)
+add_pybind11_module(operators SOURCE_FILES operators.cpp LINK_TARGETS pysrf OUTPUT_TARGET operators_target)
+add_pybind11_module(options SOURCE_FILES options.cpp LINK_TARGETS pysrf OUTPUT_TARGET options_target)
+add_pybind11_module(pipeline SOURCE_FILES pipeline.cpp LINK_TARGETS pysrf OUTPUT_TARGET pipeline_target)
+add_pybind11_module(segment SOURCE_FILES segment.cpp LINK_TARGETS pysrf OUTPUT_TARGET segment_target)
+# add_pybind11_module(segment_definition SOURCE_FILES segment_definition.cpp LINK_TARGETS pysrf OUTPUT_TARGET segment_definition_target)
```

Comment:
Is something not working with the segment_definition module?

Reply:
Its just not needed anymore and produces a compilation error